cli/command/system/prune.go

```diff
@@ -92,19 +117,19 @@ func runPrune(dockerCli command.Cli, options pruneOptions) error {
 		pruneFuncs = append(pruneFuncs, runBuildCachePrune)
 	}
 
-	var spaceReclaimed uint64
 	for _, pruneFn := range pruneFuncs {
 		spc, output, err := pruneFn(dockerCli, options.filter)
 		if err != nil {
 			return err
 		}
 		spaceReclaimed += spc
 		if output != "" {
-			fmt.Fprintln(dockerCli.Out(), output)
+			generalOutput += fmt.Sprintln(output)
```

Comment:
Instead of string concatenation it is generally better (both faster, and more obvious) to use either a `bytes.Buffer{}` with `WriteString()` or a `[]string{}` with `strings.Join(outputs, "\n")`

Reply:
I used strings.Join as a correction.